mixins/anchor-manager.js

```diff
@@ -0,0 +1,169 @@
+// AnchorManagerMixin is a React mixin that can be used to make the
+// component support permalinkable anchors (URL fragments) that
+// attract the user's attention to them when navigated to.
+//
+// Any component that uses this mixin supports an `anchorId` prop,
+// which is the URL fragment (without the leading hash) of the anchor.
+// This is expected to be constant throughout the lifetime of the
+// component. (If `anchorId` is undefined, the mixin will be largely
+// disabled.)
```

Comment:
Yeah, it's because the Clubs Toolkit--which uses the Expander component, which uses this AnchorManagerMixin--isn't using it... while we could (and probably should) add permalinks to the Clubs Toolkit sections, it didn't seem like something that the Expander component should absoutely *require* in order to use, so I made it optional.